packages/now-cli/test/helpers/prepare.js

```diff
@@ -547,34 +378,21 @@ CMD ["node", "index.js"]`,
     },
   };
 
-  for (const typeName of Object.keys(spec)) {
-    const needed = spec[typeName];
+  for (const [typeName, needed] of Object.entries(spec)) {
     const directory = join(
       __dirname,
       '..',
       'fixtures',
       'integration',
       typeName
     );
+
     await mkdirp(directory);
 
-    if (Array.isArray(needed)) {
-      // Get content from the defined files
-      for (const name of needed) {
-        const file = join(directory, name);
-        const content = files[name];
-        await mkdirp(dirname(file));
-        await writeFile(file.replace('-builds', ''), content);
-      }
-    } else {
-      // Get content from the object property
-      const names = Object.keys(needed);
-      for (const name of names) {
-        const file = join(directory, name);
-        const content = needed[name];
-        await mkdirp(dirname(file));
-        await writeFile(file.replace('-builds', ''), content);
-      }
+    for (const [name, content] of Object.entries(needed)) {
+      const file = join(directory, name);
+      await mkdirp(dirname(file));
+      await writeFile(file.replace('-builds', ''), content);
```

Comment:
Do we still need to replace `-builds`? I thought that was strange behavior before.

Reply:
I don't think so. I don't see `-builds` used anywhere.